src/mgr/MgrCommands.h

```diff
@@ -5,6 +5,8 @@
 
 // see MonCommands.h
 
+COMMAND("get_command_descriptions", "list commands descriptions", "mgr", "r", "cli,rest")
```

Comment:
well, I saw that OSD supports this command. and, I want to let CLI client could run `ceph tell mgr get_command_descriptions` at first, CLI client will return -INVAL error if we don't add this line. 

now, the question is whether we should support `tell mgr get_command_descriptions` in CLI client. IMO, we should support it so that senior user could get detail information about MGR commands.